docs/fr/api-dev.md

```diff
@@ -4,7 +4,7 @@
 
 ```bash
 cd api/
-ln -s $(pwd)/../pyalbert/pyalbert
+ln -s $(pwd)/../pyalbert
```

Comment:
Hello,
Thanks. I think it should be `ln -s $(pwd)/../pyalbert pyalbert` instead.